src/qibo/tensorflow/callbacks.py

```diff
@@ -150,25 +151,41 @@ class EntanglementEntropy(PartialTrace):
             print(entropy[:])
             # Should print [0, 0, 1] which is the entanglement entropy
             # after every gate in the calculation.
+            print(entropy.spectrum)

```

Comment:
I think in this example this will print an empty list because `compute_spectrum = False` by default. We have to pass `compute_spectrum = True` when we define the entropy callback in line 140.

Reply:
Thanks well spotted!